### ISSUE TITLE
Error when running the workflow

### ISSUE THREAD

**Issue by nomineb** [PyPSA/pypsa-usa#95]
### Checklist

- [X] I am using the current [`master`](https://github.com/PyPSA/pypsa-usa/tree/master) branch
- [X] I am running on an up-to-date [`pypsa-usa` environment](https://github.com/PyPSA/pypsa-usa/blob/master/workflow/envs/environment.yaml). Update via `conda env update -f envs/environment.yaml`

### The Issue

When I try to run the workflow on default configuration, I'm running into an error (error message in Error message section). 

### Steps To Reproduce

1. Run command: `snakemake -j 4 --configfile config/config.default.yaml`
2. Config file: config/config.default.yaml

### Expected Behavior

Completion of workflow run.

### Error Message
```
Traceback (most recent call last):
  File "/Users/nomio/Documents/REU/Fall_2324/pypsa-usa/workflow/.snakemake/scripts/tmpt25gjbej.retrieve_cost_data_usa.py", line 133, in <module>
    progress_retrieve(xlsx, save_atb_transport)
  File "/Users/nomio/Documents/REU/Fall_2324/pypsa-usa/workflow/scripts/_helpers.py", line 223, in progress_retrieve
    urllib.request.urlretrieve(url, file, reporthook=dlProgress)
  File "/Users/nomio/miniforge3/envs/pypsa-usa/lib/python3.11/urllib/request.py", line 241, in urlretrieve
    with contextlib.closing(urlopen(url, data)) as fp:
                            ^^^^^^^^^^^^^^^^^^
  File "/Users/nomio/miniforge3/envs/pypsa-usa/lib/python3.11/urllib/request.py", line 216, in urlopen
    return opener.open(url, data, timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/nomio/miniforge3/envs/pypsa-usa/lib/python3.11/urllib/request.py", line 519, in open
    response = self._open(req, data)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/nomio/miniforge3/envs/pypsa-usa/lib/python3.11/urllib/request.py", line 536, in _open
    result = self._call_chain(self.handle_open, protocol, protocol +
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/nomio/miniforge3/envs/pypsa-usa/lib/python3.11/urllib/request.py", line 496, in _call_chain
    result = func(*args)
             ^^^^^^^^^^^
  File "/Users/nomio/miniforge3/envs/pypsa-usa/lib/python3.11/urllib/request.py", line 1391, in https_open
    return self.do_open(http.client.HTTPSConnection, req,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/nomio/miniforge3/envs/pypsa-usa/lib/python3.11/urllib/request.py", line 1351, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [Errno 8] nodename nor servname provided, or not known>
100% (100 of 100) |#####################################################| Elapsed Time: 0:00:00 ETA:  00:00:00
[Thu Oct 12 15:32:37 2023]
Error in rule retrieve_cost_data_usa:
    jobid: 17
    output: resources/costs/nrel_atb.parquet, resources/costs/nrel_atb_transport.xlsx, resources/costs/ng_electric_power_price.csv, resources/costs/ng_industrial_price.csv, resources/costs/ng_commercial_price.csv, resources/costs/ng_residential_price.csv
    log: logs/retrieve_cost_data_usa.log (check log file(s) for error details)

RuleException:
CalledProcessError in file /Users/nomio/Documents/REU/Fall_2324/pypsa-usa/workflow/Snakefile, line 218:
Command 'set -euo pipefail;  /Users/nomio/miniforge3/envs/pypsa-usa/bin/python3.11 /Users/nomio/Documents/REU/Fall_2324/pypsa-usa/workflow/.snakemake/scripts/tmpt25gjbej.retrieve_cost_data_usa.py' returned non-zero exit status 1.
  File "/Users/nomio/Documents/REU/Fall_2324/pypsa-usa/workflow/Snakefile", line 218, in __rule_retrieve_cost_data_usa
  File "/Users/nomio/miniforge3/envs/pypsa-usa/lib/python3.11/concurrent/futures/thread.py", line 58, in run
Removing output files of failed job retrieve_cost_data_usa since they might be corrupted:
resources/costs/nrel_atb.parquet
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2023-10-12T153223.004632.snakemake.log
```

### Conda list
```
# packages in environment at /Users/nomio/anaconda3/envs/pypsa-usa:
#
# Name                    Version                   Build  Channel
affine                    2.4.0              pyhd8ed1ab_0    conda-forge
amply                     0.1.6              pyhd8ed1ab_0    conda-forge
anyio                     4.0.0              pyhd8ed1ab_0    conda-forge
appdirs                   1.4.4              pyh9f0ad1d_0    conda-forge
appnope                   0.1.3              pyhd8ed1ab_0    conda-forge
argon2-cffi               23.1.0             pyhd8ed1ab_0    conda-forge
argon2-cffi-bindings      21.2.0          py311heffc1b2_4    conda-forge
arrow                     1.3.0              pyhd8ed1ab_0    conda-forge
asttokens                 2.4.0              pyhd8ed1ab_0    conda-forge
async-lru                 2.0.4              pyhd8ed1ab_0    conda-forge
atlite                    0.2.11             pyhd8ed1ab_0    conda-forge
attrs                     23.1.0             pyh71513ae_1    conda-forge
aws-c-auth                0.7.4                h2ed40ff_1    conda-forge
aws-c-cal                 0.6.2                h12c43c8_2    conda-forge
aws-c-common              0.9.3                hb547adb_0    conda-forge
aws-c-compression         0.2.17               h12c43c8_3    conda-forge
aws-c-event-stream        0.3.2                h88fe898_1    conda-forge
aws-c-http                0.7.13               he062130_1    conda-forge
aws-c-io                  0.13.32              h5bd1e48_6    conda-forge
aws-c-mqtt                0.9.6                h3ffe4bb_2    conda-forge
aws-c-s3                  0.3.17               hf064be9_3    conda-forge
aws-c-sdkutils            0.1.12               h12c43c8_2    conda-forge
aws-checksums             0.1.17               h12c43c8_2    conda-forge
aws-crt-cpp               0.24.2               h6758146_0    conda-forge
aws-sdk-cpp               1.11.156             ha282cf0_4    conda-forge
babel                     2.13.0             pyhd8ed1ab_0    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                pyhd8ed1ab_3    conda-forge
backports.functools_lru_cache 1.6.5              pyhd8ed1ab_0    conda-forge
beautifulsoup4            4.12.2             pyha770c72_0    conda-forge
bleach                    6.1.0              pyhd8ed1ab_0    conda-forge
blosc                     1.21.5               hc338f07_0    conda-forge
bokeh                     3.3.0              pyhd8ed1ab_0    conda-forge
bottleneck                1.3.7           py311hb49d859_1    conda-forge
branca                    0.6.0              pyhd8ed1ab_0    conda-forge
brotli                    1.1.0                hb547adb_1    conda-forge
brotli-bin                1.1.0                hb547adb_1    conda-forge
brotli-python             1.1.0           py311ha891d26_1    conda-forge
bzip2                     1.0.8                h3422bc3_4    conda-forge
c-ares                    1.20.1               h93a5062_0    conda-forge
c-blosc2                  2.10.5               h8eb3132_0    conda-forge
ca-certificates           2023.7.22            hf0a4a13_0    conda-forge
cached-property           1.5.2                hd8ed1ab_1    conda-forge
cached_property           1.5.2              pyha770c72_1    conda-forge
cairo                     1.18.0               hd1e100b_0    conda-forge
cartopy                   0.22.0          py311h9e438b8_0    conda-forge
cdsapi                    0.6.1              pyhd8ed1ab_0    conda-forge
certifi                   2023.7.22          pyhd8ed1ab_0    conda-forge
cffi                      1.16.0          py311h4a08483_0    conda-forge
cfitsio                   4.3.0                hca87796_0    conda-forge
cftime                    1.6.2           py311hb49d859_2    conda-forge
charset-normalizer        3.3.0              pyhd8ed1ab_0    conda-forge
click                     8.1.7           unix_pyh707e725_0    conda-forge
click-plugins             1.1.1                      py_0    conda-forge
cligj                     0.7.2              pyhd8ed1ab_1    conda-forge
cloudpickle               2.2.1              pyhd8ed1ab_0    conda-forge
coin-or-cbc               2.10.10              hfaca913_0    conda-forge
coin-or-cgl               0.60.7               hf050ae7_0    conda-forge
coin-or-clp               1.17.8               h65c2c7c_0    conda-forge
coin-or-osi               0.108.8              h19bcb3e_0    conda-forge
coin-or-utils             2.11.9               h12587b1_0    conda-forge
coincbc                   2.10.10           0_metapackage    conda-forge
colorama                  0.4.6              pyhd8ed1ab_0    conda-forge
comm                      0.1.4              pyhd8ed1ab_0    conda-forge
configargparse            1.7                pyhd8ed1ab_0    conda-forge
connection_pool           0.0.3              pyhd3deb0d_0    conda-forge
contourpy                 1.1.1           py311he4fd1f5_1    conda-forge
country_converter         1.0.0              pyhd8ed1ab_1    conda-forge
countrycode               0.3.0                    pypi_0    pypi
cycler                    0.12.1             pyhd8ed1ab_0    conda-forge
cytoolz                   0.12.2          py311heffc1b2_1    conda-forge
dask                      2023.9.3           pyhd8ed1ab_0    conda-forge
dask-core                 2023.9.3           pyhd8ed1ab_0    conda-forge
datrie                    0.8.2           py311heffc1b2_7    conda-forge
debugpy                   1.8.0           py311ha891d26_1    conda-forge
decorator                 5.1.1              pyhd8ed1ab_0    conda-forge
defusedxml                0.7.1              pyhd8ed1ab_0    conda-forge
deprecation               2.1.0              pyh9f0ad1d_0    conda-forge
descartes                 1.1.0                      py_4    conda-forge
distributed               2023.9.3           pyhd8ed1ab_0    conda-forge
docutils                  0.20.1          py311h267d04e_2    conda-forge
dpath                     2.1.6              pyha770c72_0    conda-forge
entrypoints               0.4                pyhd8ed1ab_0    conda-forge
et_xmlfile                1.1.0              pyhd8ed1ab_0    conda-forge
exceptiongroup            1.1.3              pyhd8ed1ab_0    conda-forge
executing                 1.2.0              pyhd8ed1ab_0    conda-forge
expat                     2.5.0                hb7217d7_1    conda-forge
fiona                     1.9.4           py311h45231e3_1    conda-forge
folium                    0.14.0             pyhd8ed1ab_0    conda-forge
font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
font-ttf-inconsolata      3.000                h77eed37_0    conda-forge
font-ttf-source-code-pro  2.038                h77eed37_0    conda-forge
font-ttf-ubuntu           0.83                 hab24e00_0    conda-forge
fontconfig                2.14.2               h82840c6_0    conda-forge
fonts-conda-ecosystem     1                             0    conda-forge
fonts-conda-forge         1                             0    conda-forge
fonttools                 4.43.1          py311h05b510d_0    conda-forge
fqdn                      1.5.1              pyhd8ed1ab_0    conda-forge
freetype                  2.12.1               hadb7bae_2    conda-forge
freexl                    2.0.0                hfbad9fb_0    conda-forge
fs                        2.4.11             pyhd8ed1ab_3    conda-forge
fsspec                    2023.9.2           pyh1a96a4e_0    conda-forge
gdal                      3.7.2           py311h32a4f3d_6    conda-forge
geographiclib             1.52               pyhd8ed1ab_0    conda-forge
geopandas                 0.14.0             pyhd8ed1ab_1    conda-forge
geopandas-base            0.14.0             pyha770c72_1    conda-forge
geopy                     2.4.0              pyhd8ed1ab_0    conda-forge
geos                      3.12.0               h13dd4ca_0    conda-forge
geotiff                   1.7.1               h71398c0_14    conda-forge
gettext                   0.21.1               h0186832_0    conda-forge
gflags                    2.2.2             hc88da5d_1004    conda-forge
giflib                    5.2.1                h1a8c8d9_3    conda-forge
gitdb                     4.0.10             pyhd8ed1ab_0    conda-forge
gitpython                 3.1.37             pyhd8ed1ab_0    conda-forge
glog                      0.6.0                h6da1cb0_0    conda-forge
gurobipy                  10.0.3                   pypi_0    pypi
hdf4                      4.2.15               h2ee6834_7    conda-forge
hdf5                      1.14.2          nompi_h3aba7b3_100    conda-forge
highspy                   1.5.3                    pypi_0    pypi
humanfriendly             10.0               pyhd8ed1ab_6    conda-forge
icu                       73.2                 hc8870d7_0    conda-forge
idna                      3.4                pyhd8ed1ab_0    conda-forge
importlib-metadata        6.8.0              pyha770c72_0    conda-forge
importlib_metadata        6.8.0                hd8ed1ab_0    conda-forge
importlib_resources       6.1.0              pyhd8ed1ab_0    conda-forge
iniconfig                 2.0.0              pyhd8ed1ab_0    conda-forge
ipykernel                 6.25.2             pyh1050b4e_0    conda-forge
ipython                   8.16.1             pyh31c8845_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
ipywidgets                8.1.1              pyhd8ed1ab_0    conda-forge
isoduration               20.11.0            pyhd8ed1ab_0    conda-forge
jedi                      0.19.1             pyhd8ed1ab_0    conda-forge
jinja2                    3.1.2              pyhd8ed1ab_1    conda-forge
joblib                    1.3.2              pyhd8ed1ab_0    conda-forge
json-c                    0.17                 h40ed0f5_0    conda-forge
json5                     0.9.14             pyhd8ed1ab_0    conda-forge
jsonpointer               2.4             py311h267d04e_3    conda-forge
jsonschema                4.19.1             pyhd8ed1ab_0    conda-forge
jsonschema-specifications 2023.7.1           pyhd8ed1ab_0    conda-forge
jsonschema-with-format-nongpl 4.19.1             pyhd8ed1ab_0    conda-forge
jupyter                   1.0.0             pyhd8ed1ab_10    conda-forge
jupyter-lsp               2.2.0              pyhd8ed1ab_0    conda-forge
jupyter_client            8.4.0              pyhd8ed1ab_0    conda-forge
jupyter_console           6.6.3              pyhd8ed1ab_0    conda-forge
jupyter_core              5.4.0           py311h267d04e_0    conda-forge
jupyter_events            0.7.0              pyhd8ed1ab_2    conda-forge
jupyter_server            2.7.3              pyhd8ed1ab_1    conda-forge
jupyter_server_terminals  0.4.4              pyhd8ed1ab_1    conda-forge
jupyterlab                4.0.6              pyhd8ed1ab_0    conda-forge
jupyterlab_pygments       0.2.2              pyhd8ed1ab_0    conda-forge
jupyterlab_server         2.25.0             pyhd8ed1ab_0    conda-forge
jupyterlab_widgets        3.0.9              pyhd8ed1ab_0    conda-forge
kealib                    1.5.2                h47b5e36_1    conda-forge
kiwisolver                1.4.5           py311he4fd1f5_1    conda-forge
krb5                      1.21.2               h92f50d5_0    conda-forge
lcms2                     2.15                 hf2736f0_3    conda-forge
lerc                      4.0.0                h9a09cb3_0    conda-forge
libabseil                 20230802.1      cxx17_h13dd4ca_0    conda-forge
libaec                    1.1.2                h13dd4ca_1    conda-forge
libarchive                3.7.2                h82b9b87_0    conda-forge
libarrow                  13.0.0           h5325880_8_cpu    conda-forge
libblas                   3.9.0           18_osxarm64_openblas    conda-forge
libboost-headers          1.82.0               hce30654_6    conda-forge
libbrotlicommon           1.1.0                hb547adb_1    conda-forge
libbrotlidec              1.1.0                hb547adb_1    conda-forge
libbrotlienc              1.1.0                hb547adb_1    conda-forge
libcblas                  3.9.0           18_osxarm64_openblas    conda-forge
libcrc32c                 1.1.2                hbdafb3b_0    conda-forge
libcurl                   8.4.0                h2d989ff_0    conda-forge
libcxx                    16.0.6               h4653b0c_0    conda-forge
libdeflate                1.19                 hb547adb_0    conda-forge
libedit                   3.1.20191231         hc8eb9b7_2    conda-forge
libev                     4.33                 h642e427_1    conda-forge
libevent                  2.1.12               h2757513_1    conda-forge
libexpat                  2.5.0                hb7217d7_1    conda-forge
libffi                    3.4.2                h3422bc3_5    conda-forge
libgdal                   3.7.2                h40a65a3_6    conda-forge
libgfortran               5.0.0           13_2_0_hd922786_1    conda-forge
libgfortran5              13.2.0               hf226fd6_1    conda-forge
libglib                   2.78.0               h24e9cb9_0    conda-forge
libgoogle-cloud           2.12.0               h5a37b55_3    conda-forge
libgrpc                   1.58.1               h941e00e_0    conda-forge
libiconv                  1.17                 he4db4b2_0    conda-forge
libjpeg-turbo             3.0.0                hb547adb_1    conda-forge
libkml                    1.3.0             h1eb4d9f_1018    conda-forge
liblapack                 3.9.0           18_osxarm64_openblas    conda-forge
liblapacke                3.9.0           18_osxarm64_openblas    conda-forge
libnetcdf                 4.9.2           nompi_hb2fb864_112    conda-forge
libnghttp2                1.52.0               hae82a92_0    conda-forge
libopenblas               0.3.24          openmp_hd76b1f2_0    conda-forge
libpng                    1.6.39               h76d750c_0    conda-forge
libpq                     16.0                 hcea71ed_1    conda-forge
libprotobuf               4.24.3               hf590ac1_0    conda-forge
librttopo                 1.1.0               h667cd51_14    conda-forge
libsodium                 1.0.18               h27ca646_1    conda-forge
libspatialindex           1.9.3                hbdafb3b_4    conda-forge
libspatialite             5.1.0                h32510b6_0    conda-forge
libsqlite                 3.43.2               h091b4b1_0    conda-forge
libssh2                   1.11.0               h7a5bd25_0    conda-forge
libthrift                 0.19.0               h026a170_1    conda-forge
libtiff                   4.6.0                ha8a6c65_2    conda-forge
libutf8proc               2.8.0                h1a8c8d9_0    conda-forge
libwebp-base              1.3.2                hb547adb_0    conda-forge
libxcb                    1.15                 hf346824_0    conda-forge
libxml2                   2.11.5               h25269f3_1    conda-forge
libxslt                   1.1.37               h1728932_1    conda-forge
libzip                    1.10.1               ha0bc3c6_3    conda-forge
libzlib                   1.2.13               h53f4e23_5    conda-forge
linopy                    0.2.6              pyhd8ed1ab_0    conda-forge
llvm-openmp               17.0.2               h1c12783_0    conda-forge
locket                    1.0.0              pyhd8ed1ab_0    conda-forge
lxml                      4.9.3           py311hbafe683_1    conda-forge
lz4                       4.3.2           py311h0c6baac_1    conda-forge
lz4-c                     1.9.4                hb7217d7_0    conda-forge
lzo                       2.10              h642e427_1000    conda-forge
mapclassify               2.6.1              pyhd8ed1ab_0    conda-forge
markupsafe                2.1.3           py311heffc1b2_1    conda-forge
matplotlib                3.8.0           py311ha1ab1f8_2    conda-forge
matplotlib-base           3.8.0           py311hfdba5f6_2    conda-forge
matplotlib-inline         0.1.6              pyhd8ed1ab_0    conda-forge
memory_profiler           0.61.0             pyhd8ed1ab_0    conda-forge
minizip                   4.0.1                h5a7aac7_5    conda-forge
mistune                   3.0.1              pyhd8ed1ab_0    conda-forge
msgpack-python            1.0.6           py311he4fd1f5_0    conda-forge
munch                     4.0.0              pyhd8ed1ab_0    conda-forge
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
nbclient                  0.8.0              pyhd8ed1ab_0    conda-forge
nbconvert                 7.9.2              pyhd8ed1ab_0    conda-forge
nbconvert-core            7.9.2              pyhd8ed1ab_0    conda-forge
nbconvert-pandoc          7.9.2              pyhd8ed1ab_0    conda-forge
nbformat                  5.9.2              pyhd8ed1ab_0    conda-forge
ncurses                   6.4                  h7ea286d_0    conda-forge
nest-asyncio              1.5.7              pyhd8ed1ab_0    conda-forge
netcdf4                   1.6.4           nompi_py311hbb54521_103    conda-forge
networkx                  3.1                pyhd8ed1ab_0    conda-forge
notebook                  7.0.4              pyhd8ed1ab_0    conda-forge
notebook-shim             0.2.3              pyhd8ed1ab_0    conda-forge
nspr                      4.35                 hb7217d7_0    conda-forge
nss                       3.94                 hc6b9969_0    conda-forge
numexpr                   2.8.7           py311h6e08293_3    conda-forge
numpy                     1.26.0          py311hb8f3215_0    conda-forge
openjdk                   20.0.2               hbe7ddab_2    conda-forge
openjpeg                  2.5.0                h4c1507b_3    conda-forge
openpyxl                  3.1.2           py311heffc1b2_0    conda-forge
openssl                   3.1.3                h53f4e23_0    conda-forge
orc                       1.9.0                hcd02cb2_3    conda-forge
overrides                 7.4.0              pyhd8ed1ab_0    conda-forge
packaging                 23.2               pyhd8ed1ab_0    conda-forge
pandas                    2.0.3           py311h9e438b8_1    conda-forge
pandoc                    3.1.3                hce30654_0    conda-forge
pandocfilters             1.5.0              pyhd8ed1ab_0    conda-forge
parso                     0.8.3              pyhd8ed1ab_0    conda-forge
partd                     1.4.1              pyhd8ed1ab_0    conda-forge
patsy                     0.5.3              pyhd8ed1ab_0    conda-forge
pcre2                     10.40                hb34f9b4_0    conda-forge
pexpect                   4.8.0              pyh1a96a4e_2    conda-forge
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    10.0.1          py311h8dc27b9_2    conda-forge
pip                       23.2.1             pyhd8ed1ab_0    conda-forge
pixman                    0.42.2               h13dd4ca_0    conda-forge
pkgutil-resolve-name      1.3.10             pyhd8ed1ab_1    conda-forge
plac                      1.4.0              pyhd8ed1ab_0    conda-forge
platformdirs              3.11.0             pyhd8ed1ab_0    conda-forge
pluggy                    1.3.0              pyhd8ed1ab_0    conda-forge
ply                       3.11                       py_1    conda-forge
polars                    0.18.15                  pypi_0    pypi
poppler                   23.08.0              hcdd998b_3    conda-forge
poppler-data              0.4.12               hd8ed1ab_0    conda-forge
postgresql                16.0                 h00cd704_1    conda-forge
progressbar2              4.2.0              pyhd8ed1ab_0    conda-forge
proj                      9.3.0                h4b7dc33_1    conda-forge
prometheus_client         0.17.1             pyhd8ed1ab_0    conda-forge
prompt-toolkit            3.0.39             pyha770c72_0    conda-forge
prompt_toolkit            3.0.39               hd8ed1ab_0    conda-forge
psutil                    5.9.5           py311heffc1b2_1    conda-forge
pthread-stubs             0.4               h27ca646_1001    conda-forge
ptyprocess                0.7.0              pyhd3deb0d_0    conda-forge
pulp                      2.7.0           py311h267d04e_1    conda-forge
pure_eval                 0.2.2              pyhd8ed1ab_0    conda-forge
py-cpuinfo                9.0.0              pyhd8ed1ab_0    conda-forge
pyarrow                   13.0.0          py311hd7bc329_8_cpu    conda-forge
pycountry                 22.3.5             pyhd8ed1ab_0    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pygments                  2.16.1             pyhd8ed1ab_0    conda-forge
pyobjc-core               10.0            py311hb702dc4_0    conda-forge
pyobjc-framework-cocoa    10.0            py311hb702dc4_1    conda-forge
pyomo                     6.6.2           py311ha891d26_1    conda-forge
pyparsing                 3.1.1              pyhd8ed1ab_0    conda-forge
pyproj                    3.6.1           py311he1f72a7_2    conda-forge
pypsa                     0.25.1             pyhd8ed1ab_0    conda-forge
pyshp                     2.3.1              pyhd8ed1ab_0    conda-forge
pysocks                   1.7.1              pyha2e5f31_6    conda-forge
pytables                  3.9.1           py311hce229b4_0    conda-forge
pytest                    7.4.2              pyhd8ed1ab_0    conda-forge
python                    3.11.6          h47c9636_0_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-fastjsonschema     2.18.1             pyhd8ed1ab_0    conda-forge
python-json-logger        2.0.7              pyhd8ed1ab_0    conda-forge
python-tzdata             2023.3             pyhd8ed1ab_0    conda-forge
python-utils              3.8.1              pyhd8ed1ab_0    conda-forge
python_abi                3.11                    4_cp311    conda-forge
pytz                      2023.3.post1       pyhd8ed1ab_0    conda-forge
pyyaml                    6.0.1           py311heffc1b2_1    conda-forge
pyzmq                     25.1.1          py311hb1af645_1    conda-forge
qtconsole-base            5.4.4              pyha770c72_0    conda-forge
qtpy                      2.4.0              pyhd8ed1ab_0    conda-forge
rasterio                  1.3.8           py311h62dc74c_3    conda-forge
re2                       2023.03.02           hc5e2d97_0    conda-forge
readline                  8.2                  h92ec313_1    conda-forge
referencing               0.30.2             pyhd8ed1ab_0    conda-forge
requests                  2.31.0             pyhd8ed1ab_0    conda-forge
reretry                   0.11.8             pyhd8ed1ab_0    conda-forge
rfc3339-validator         0.1.4              pyhd8ed1ab_0    conda-forge
rfc3986-validator         0.1.1              pyh9f0ad1d_0    conda-forge
rpds-py                   0.10.4          py311h94f323b_0    conda-forge
rtree                     1.0.1           py311hd698ff7_3    conda-forge
scikit-learn              1.3.1           py311ha25ca4d_1    conda-forge
scipy                     1.11.3          py311h93d07a4_1    conda-forge
seaborn                   0.13.0               hd8ed1ab_0    conda-forge
seaborn-base              0.13.0             pyhd8ed1ab_0    conda-forge
send2trash                1.8.2              pyhd1c38e8_0    conda-forge
setuptools                68.2.2             pyhd8ed1ab_0    conda-forge
setuptools-scm            8.0.4              pyhd8ed1ab_0    conda-forge
setuptools_scm            8.0.4                hd8ed1ab_0    conda-forge
shapely                   2.0.1           py311h7089352_3    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
smart_open                6.4.0              pyhd8ed1ab_0    conda-forge
smmap                     3.0.5              pyh44b312d_0    conda-forge
snakemake-minimal         7.32.4             pyhdfd78af_0    bioconda
snappy                    1.1.10               h17c5cce_0    conda-forge
sniffio                   1.3.0              pyhd8ed1ab_0    conda-forge
snuggs                    1.4.7                      py_0    conda-forge
sortedcontainers          2.4.0              pyhd8ed1ab_0    conda-forge
soupsieve                 2.5                pyhd8ed1ab_1    conda-forge
sqlite                    3.43.2               hf2abe2d_0    conda-forge
stack_data                0.6.2              pyhd8ed1ab_0    conda-forge
statsmodels               0.14.0          py311h9ea6feb_2    conda-forge
stopit                    1.1.2                      py_0    conda-forge
tabula-py                 2.7.0           py311h267d04e_1    conda-forge
tabulate                  0.9.0              pyhd8ed1ab_1    conda-forge
tblib                     2.0.0              pyhd8ed1ab_0    conda-forge
terminado                 0.17.1             pyhd1c38e8_0    conda-forge
threadpoolctl             3.2.0              pyha21a80b_0    conda-forge
throttler                 1.2.2              pyhd8ed1ab_0    conda-forge
tiledb                    2.16.3               he15c4da_3    conda-forge
tinycss2                  1.2.1              pyhd8ed1ab_0    conda-forge
tk                        8.6.13               hb31c410_0    conda-forge
tomli                     2.0.1              pyhd8ed1ab_0    conda-forge
toolz                     0.12.0             pyhd8ed1ab_0    conda-forge
toposort                  1.10               pyhd8ed1ab_0    conda-forge
tornado                   6.3.3           py311heffc1b2_1    conda-forge
tqdm                      4.66.1             pyhd8ed1ab_0    conda-forge
traitlets                 5.11.2             pyhd8ed1ab_0    conda-forge
tsam                      2.3.1                    pypi_0    pypi
types-python-dateutil     2.8.19.14          pyhd8ed1ab_0    conda-forge
typing-extensions         4.8.0                hd8ed1ab_0    conda-forge
typing_extensions         4.8.0              pyha770c72_0    conda-forge
typing_utils              0.1.0              pyhd8ed1ab_0    conda-forge
tzcode                    2023c                h1a8c8d9_0    conda-forge
tzdata                    2023c                h71feb2d_0    conda-forge
uri-template              1.3.0              pyhd8ed1ab_0    conda-forge
uriparser                 0.9.7                hb7217d7_1    conda-forge
urllib3                   2.0.6              pyhd8ed1ab_0    conda-forge
validators                0.22.0             pyhd8ed1ab_0    conda-forge
vresutils                 0.3.1                    pypi_0    pypi
wcwidth                   0.2.8              pyhd8ed1ab_0    conda-forge
webcolors                 1.13               pyhd8ed1ab_0    conda-forge
webencodings              0.5.1              pyhd8ed1ab_2    conda-forge
websocket-client          1.6.4              pyhd8ed1ab_0    conda-forge
wheel                     0.41.2             pyhd8ed1ab_0    conda-forge
widgetsnbextension        4.0.9              pyhd8ed1ab_0    conda-forge
wrapt                     1.15.0          py311heffc1b2_1    conda-forge
xarray                    2023.9.0           pyhd8ed1ab_0    conda-forge
xerces-c                  3.2.4                hd886eac_3    conda-forge
xlrd                      2.0.1              pyhd8ed1ab_3    conda-forge
xorg-libxau               1.0.11               hb547adb_0    conda-forge
xorg-libxdmcp             1.1.3                h27ca646_0    conda-forge
xyzservices               2023.10.0          pyhd8ed1ab_0    conda-forge
xz                        5.2.6                h57fd34a_0    conda-forge
yaml                      0.2.5                h3422bc3_2    conda-forge
yte                       1.5.1              pyha770c72_2    conda-forge
zeromq                    4.3.4                hbdafb3b_1    conda-forge
zict                      3.0.0              pyhd8ed1ab_0    conda-forge
zipp                      3.17.0             pyhd8ed1ab_0    conda-forge
zlib                      1.2.13               h53f4e23_5    conda-forge
zlib-ng                   2.0.7                h1a8c8d9_0    conda-forge
zstd                      1.5.5                h4f39d0f_0    conda-forge
```

**Comment by ktehranchi** [comment]
Hi Nomin... I haven't seen this before. Three steps to try:

- Rerun workflow from scratch `snakemake -j2 -F`
- Could you run this with snakemake -j2 without the config specified in your snakemake call? 
- can you post your   ` logs/western/build_renewable_profile_solar.log` file?

**Comment by trevorb1** [comment]
I just tried on my side and I got the different error. Looks like its being thrown when trying to download transport cost data. I will look at this right now! 

```bash
<urlopen error [Errno -3] Temporary failure in name resolution>
socket.gaierror: [Errno -3] Temporary failure in name resolution

During handling of the above exception, another exception occurred:

  File "/home/trevor/repos/pypsa-usa/workflow/scripts/_helpers.py", line 223, in progress_retrieve
    urllib.request.urlretrieve(url, file, reporthook=dlProgress)
  File "/home/trevor/repos/pypsa-usa/workflow/scripts/retrieve_cost_data_usa.py", line 129, in <module>
    progress_retrieve(xlsx, save_atb_transport)
urllib.error.URLError: <urlopen error [Errno -3] Temporary failure in name resolution>
```

**Comment by trevorb1** [comment]
Closed with PR #98

**Comment by trevorb1** [comment]
Thanks for flagging this @nomineb!
